test-requirements.txt

```diff
@@ -1,33 +1,11 @@
-attrs                           # via pytest
-certifi                         # via requests
-chardet                         # via requests
-cityhash                        # via katdal
+-c https://raw.githubusercontent.com/ska-sa/katsdpdockerbase/master/docker-base-build/base-requirements.txt
+-c requirements.txt
+
 coverage
-dask                            # via katdal
-ephem                           # via pyephem
-future                          # via katdal, katpoint
-h5py                            # via katdal
-idna                            # via requests
-importlib-metadata              # via pytest, pluggy
 katpoint==0.9                   # via katdal
```

Comment:
Any reason for not using the GitHub version?

Reply:
You'd have to ask past-me. Possibly he was clairvoyant enough to know that the next katpoint would be backwards-incompatible? :smile: 

It probably makes sense to use the Github version just for consistency with other packages, so I'll do that.